rticonnextdds-connector.go

```diff
@@ -372,69 +387,77 @@ func (instance *Instance) SetRune(fieldName string, value rune) error {
 	return nil
 }
 
+// SetBoolean is a function to set boolean to a fieldname of the samples
 func (instance *Instance) SetBoolean(fieldName string, value bool) error {
 	fieldNameCStr := C.CString(fieldName)
 	defer C.free(unsafe.Pointer(fieldNameCStr))
 
-	var int_value int
+	var intValue int
 	if value == true {
-		int_value = 1
+		intValue = 1
 	} else {
-		int_value = 0
+		intValue = 0
 	}
-	C.RTIDDSConnector_setBooleanIntoSamples(unsafe.Pointer(instance.output.connector.native), instance.output.nameCStr, fieldNameCStr, C.int(int_value))
+	C.RTIDDSConnector_setBooleanIntoSamples(unsafe.Pointer(instance.output.connector.native), instance.output.nameCStr, fieldNameCStr, C.int(intValue))
 	return nil
 }
 
-func (instance *Instance) SetJson(json []byte) error {
+// SetJSON is a function to set JSON string in the form of slice of bytes into Instance
+func (instance *Instance) SetJSON(json []byte) error {
 	jsonCStr := C.CString(string(json))
 	defer C.free(unsafe.Pointer(jsonCStr))
 
 	C.RTIDDSConnector_setJSONInstance(unsafe.Pointer(instance.output.connector.native), instance.output.nameCStr, jsonCStr)
 	return nil
 }
 
-// TODO - think about a new name for this function (e.g. SetType, SetFromType, FromType)
+// Set is a function that consumes an interface
+// of multiple samples with different types and value
+// TODO - think about a new name for this a function (e.g. SetType, SetFromType, FromType)
 func (instance *Instance) Set(v interface{}) (err error) {
 	jsonData, err := json.Marshal(v)
 	if err != nil {
 		return err
 	}
 
-	err = instance.SetJson(jsonData)
+	err = instance.SetJSON(jsonData)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
+// Read is a function to initialize to read the DDS messages
```

Comment:
Read is a function to read DDS samples from the DDS DataReader and allow access them via the Connector Samples. The Read function does not remove DDS samples from the DDS DataReader's receive queue.